### DISCUSSION TITLE
请问 segment 如何垂直展示呢

### DISCUSSION THREAD

**Discussion by yuruotong1** [ant-design/ant-design#38761] [Q&A] [answered]
` <Segmented
      options={[
        {
          label: (
            <div style={{ padding: 4 }}>
              <Avatar src="https://joeschmoe.io/api/v1/random" />
              <div>User 1</div>
            </div>
          ),
          value: 'user1',
        },
        {
          label: (
            <div style={{ padding: 4 }}>
              <Avatar style={{ backgroundColor: '#f56a00' }}>K</Avatar>
              <div>User 2</div>
            </div>
          ),
          value: 'user2',
        },
        {
          label: (
            <div style={{ padding: 4 }}>
              <Avatar style={{ backgroundColor: '#87d068' }} icon={<UserOutlined />} />
              <div>User 3</div>
            </div>
          ),
          value: 'user3',
        },
      ]}
    />`

我尝试了一些参数，发现只能水平展示，不能垂直展示 ！如何设置才能垂直展示呢？

![image](https://user-images.githubusercontent.com/31992251/202912423-fed4b926-997b-4378-9706-9f672f99cff7.png)

**Comment by arvinxx** [comment, marked as answer]
我在设计上基本上没见过 Segmented 竖着的。

**Reply by yuruotong1** [reply]
你可以理解为有 n  个场景，用户可以增加或选择一个场景进行操作，如果横着场景会越来越多占满一行

![image](https://user-images.githubusercontent.com/31992251/202912411-50fb1dd7-5445-404f-ab88-fb8261c61fa7.png)

**Reply by arvinxx** [reply]
那你不应该用 segmented，menu 更加合适
